Makefile

```diff
@@ -130,11 +130,13 @@ bench: fmt
 test:
 	go test -short -tags='debug testing netgo' -timeout=5s $(pkgs) -run=. -count=$(count)
 
+# Having this in place keeps the "testing interface" of the service compatible with accounts, blocker and so on.
 test-long: lint lint-ci
-	@mkdir -p cover
```

Comment:
Now it looks like `test-long` isn't running any tests which is very misleading.

Reply:
Removed.